tests/test_openjdk_devel.py

```diff
@@ -5,14 +5,28 @@
 import pytest
 
 from bci_tester.data import OPENJDK_DEVEL_11_CONTAINER
+from bci_tester.data import OPENJDK_DEVEL_17_CONTAINER
 
-
-CONTAINER_IMAGES = [OPENJDK_DEVEL_11_CONTAINER]
+CONTAINER_IMAGES = [
+    OPENJDK_DEVEL_11_CONTAINER,
+    OPENJDK_DEVEL_17_CONTAINER,
+]
 
 
 @pytest.mark.parametrize(
     "container,java_version",
-    [(OPENJDK_DEVEL_11_CONTAINER, "11")],
+    [
+        pytest.param(
+            OPENJDK_DEVEL_11_CONTAINER,
+            "11",
+            marks=OPENJDK_DEVEL_11_CONTAINER.marks,
+        ),
+        pytest.param(
+            OPENJDK_DEVEL_17_CONTAINER,
+            "17",
+            marks=OPENJDK_DEVEL_17_CONTAINER.marks,
```

Comment:
you will need to add an xfail here, see: https://github.com/SUSE/BCI-tests/blob/04b6e72f49b8690b8f54a0a53cd8d7f75b53b4ba/tests/test_openjdk_devel.py#L17